src/form/Inputs.js

```diff
@@ -54,12 +57,11 @@ export const buildCheckboxValidator = mustBe =>
 export const buildPassphraseValidator = () => buildValidator([]);
```

Comment:
Maybe a bit off-topic, but, didn't we have a `lib/validator`? Shouldn't all of these be in there?

Reply:
(fixed before I even reviewed it, truly the fastest dev (^:)